verify.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+function validate() {
+    python_version=$1
+
+    echo "Validating version $python_version..."
+
+    tag=py-ipfs-http-client-verify:$python_version
+
+    docker build --build-arg PYTHON_VERSION=$python_version -t $tag .
+    docker run -it -v $(pwd):/source:ro -w /usr/src/app $tag tox -e styleck -e typeck
```

Comment:
Please always add double-quotes (") around variable/process substitutions (ie: `$tag` → `"${tag}"`) unless you explicitly intend to invoke shell word splitting. Figuring out whitespace issues with shell is a huge PITA otherwise.